src/compiler/build/build-stats.ts

```diff
@@ -166,7 +168,15 @@ function getComponentsFileMap(config: d.Config, buildCtx: d.BuildCtx) {
   });
 }
 
-function getCollections(config: d.Config, buildCtx: d.BuildCtx) {
+function getCollections(
+  config: d.ValidatedConfig,
+  buildCtx: d.BuildCtx
+): {
+  name: string;
+  source: string;
+  tags: string[];
+}[] {
+  // @ts-ignore
```

Comment:
Just curious, why do we need to ignore here?

Reply:
tbh I have absolutely no idea, I opened this PR in November 🤣

but I removed it and looks like I probably left it in by accident